packages/Filter/src/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+import './filter.scss';
+
+export interface FilterProps extends React.ComponentProps<'button'> {
+  /** Add the main text/icon to the filter */
+  children: React.ReactNode;
+  /** Add class names to the filter */
+  className?: string;
+  /** Count of items that match the filter */
+  count?: number;
```

Comment:
why not just pass the **count** via the `children` by the user?

`<Filter>Assigned to me (1)</Filter>`

Reply:
It's a possibility. If the count is updated, will it re-render just the count and not the whole child element?

Reply:
if the number changes the `Filter` component will be re-rendered